src/parse/kdbx4.rs

```diff
@@ -1,6 +1,7 @@
 use std::convert::TryFrom;
 use std::convert::TryInto;
 
+use crate::meta::BinaryAttachment;
```

Comment:
@sseemayer nitpicking here, but why not merge that import with the import block below?